@here/harp-mapview/lib/DataSource.ts

```diff
@@ -293,6 +298,20 @@ export abstract class DataSource extends THREE.EventDispatcher {
         this.m_maxZoomLevel = level;
     }
 
+    /**
+     * Maximum geometry height above ground level this `DataSource` can produce.
+     *
+     * Used in first stage of frustum culling before [[Tile.maxGeometryHeight]] data is available.
+     *
+     * @default [[EarthConstants.MAX_BUILDING_HEIGHT]].
+     */
+    get maxGeometryHeight() {
+        return this.m_maxGeometryHeight;
```

Comment:
I think the default should be 0 and only for OmvDataSource it should be MAX_BUILDING_HEIGHT by default,
especially since the default for Tile.maxBuildingHeight is 0 as well.

Reply:
ok